src/components/actions/getApplicationData.ts

```diff
@@ -56,10 +56,20 @@ export const getApplicationData = async (input: {
       }
     : {}
 
+  const Emailconfig = {
+    host: 'server.msupply.foundation',
+    port: 465,
+    secure: true,
+    user: 'irims-dev@sussol.net',
+    defaultFromName: 'Conforma',
+    defaultFromEmail: 'test-new-noreply@email.com',
+  }
+
```

Comment:
Yeah, good, except this needs to be in "preferences.json" rather than hard-coded in here.

You'll also need to make sure it fails gracefully if the preference is missing. I made a suggestion in the original issue as to what probably should happen.